.github/mergify.yml

```diff
@@ -54,3 +54,11 @@ pull_request_rules:
       backport:
         branches:
           - v24.x
+  - name: backport patches to v25.x branch
+    conditions:
+      - base=v24.x
+      - label=A:backport/v25.x
+    actions:
+      backport:
+        branches:
+          - v25.x
```

Comment:
Ahh that's why it didn't work for me

I saw this present on `v24.x` but did not check `v25.x`

Thanks for adding